### PR TITLE
nco: update to 4.9.3

### PR DESCRIPTION
**Pull request by remkos**
* Updated version number
* Updated checksum
* Removed no longer available documentation

###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.15.5
Xcode 11.5

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: science/nco/Portfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           compilers 1.0
 PortGroup           github 1.0
 
-github.setup        nco nco 4.9.2
+github.setup        nco nco 4.9.3
 revision            1
```
Comment (mf2k) [inline science/nco/Portfile:8]: The `revision` needs to be set back to 0 when increasing the `version`.

Comment (remkos) [reply science/nco/Portfile]: Sorry, forgot. Done

**Comment by remkos** [comment]
Squashed into one commit. Finally figured out how to do that.

**Comment by reneeotten** [comment]
> Squashed into one commit. Finally figured out how to do that.

thanks, please update the commit message as well. Additional comments besides the title should only be used if the information is helpful - right now none of it really is. For example, the first three things you list are pretty obvious if you update to a newer version, so these should be removed.

**Comment by remkos** [comment]
> thanks, please update the commit message as well. Additional comments besides the title should only be used if the information is helpful - right now none of it really is. For example, the first three things you list are pretty obvious if you update to a newer version, so these should be removed.

Done as requested by @reneeotten

**Comment by reneeotten** [comment]
merging under `maintainer timeout`

**Pull request merged**